packages/translations-cli/src/index.ts

```diff
@@ -14,31 +14,43 @@ program
   .arguments('<SOURCE_DIR> <TARGET_FILE>')
   .description('', {
     SOURCE_DIR: 'dir with translation file',
-    TARGET_FILE: 'translation file to merge with',
+    TARGET_FILE: 'translation file to merge with(url or local file)',
   })
   .action((SOURCE_DIR: string, TARGET_FILE: string) => {
     try {
       const sourceData = JSON.parse(
         fs.readFileSync(`${SOURCE_DIR}/translation.json`)
       );
       let targetData = '';
-      https.get(TARGET_FILE, (resp: any) => {
-        resp.on('data', (chunk: any) => {
-          targetData += chunk;
-        });
+      if (TARGET_FILE.includes('http')) {
```

Comment:
What if file is hosed on domain with SSL and use `https` ?

Reply:
This is why I check if `TARGET_FILE` includes `http` because it will be true for both: `http` and `https`. Both of those strings contains `http`